README.md

```diff
@@ -87,15 +87,33 @@ instance of the adage-server.
    ```
    docker-compose exec core python manage.py <management command> <arguments>
    ```
+
+   Note that the files that you will pass as some of these `<arguments>`
+   will be located in the `/srv/data/` folder, as specified by the `docker cp`
+   command above.
+
+   For example, if you wanted to copy a file on your computer called
+   'SomeOrganismAnnotation.tsv', and then load it into the instance's
+   database using the `import_data` management command, you would enter:
+
+   ```
+   docker cp \
+       <local path to file>/SomeOrganismAnnotation.tsv \
+       adage-django:/srv/data/
+
+   docker-compose exec core python manage.py import_data \
+       /srv/data/SomeOrganismAnnotation.tsv
+   ```
+
    These are the management commands currently available to load data files:
    `add_ml_model`,
    `create_or_update_participation_type`,
    `delete_participation_type`,
    `import_activity`,
    `import_data`,
    `import_gene_network.py`,
-   `import_gene_sample_expr`,
-   `import_signature_gene_network`
+   `import_gene_sample_expr`, and
+   `import_signature_gene_network`.
```

Comment:
Do you think it will be more readable if we list these commands with bullets?

Reply:
Yeah - I was wondering that as well. That makes me think that bullet points might be the right approach. I'll give put them in and see how it looks.